pontoon/base/migrations/0136_index_on_translationmemoryentry.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2018-11-05 23:12
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('base', '0135_project_project_configuration'),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            "CREATE INDEX tm_locale_char_length_idx ON base_translationmemoryentry(locale_id, char_length(source))",
+            "DROP INDEX tm_locale_char_length_idx"
+        )
+    ]
```

Comment:
Can we create this index via Django? It's not very discoverable when created with a migration.